prow/cmd/status-reconciler/README.md

```diff
@@ -12,6 +12,10 @@ When the set of blocking presubmits changes for a repository, one of three cases
 The `status-reconciler` watches the job configuration for Prow and ensures that the above actions
 are taken as necessary.
 
+To exclude repos from being reconciled, passing flag `--denylist`, this can be done repeatedly.
+This is useful when moving a repo from prow instance A to prow instance B, while unwinding jobs from
+prow instance A, the jobs are not expected to be blindly lablled succeed by prow instance A.
```

Comment:
Under what circumstances are job `blindly lablled succeed`?

Reply:
Based on my understanding, when a required job is deleted, it's github context is marked as succeed by statusreconciler